doc/source/frequently-asked-questions.rst

```diff
@@ -54,46 +54,61 @@ How do I ignore words/lines/files in automatic spelling checks in pre-commit?
 Project setup
 -------------
 
-I read ``scikit-package`` supports a namespace package. What is it and how do I set it up?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+I read ``scikit-package`` allows namespace support for importing packages. What is it, and how do I set it up?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In plain English, a namespace package refers to a package that is organized in a way that allows importing and installing a package like ``pip install diffpy.pdffit2``. The package starts with an ``org-name`` or similar, and each package has a separate repository, i.e., https://github.com/diffpy/diffpy.pdffit2. The consistency in naming can be beneficial for research groups by organizing research software projects and maintaining branding, as demonstrated by software projects in https://www.diffpy.org/.
+A research group often maintains multiple software packages under a single GitHub organization.
+For the purposes of branding and also differentiating packages with similar names, it can sometimes be beneficial for the org name (or some other branding name) to appear in the package name itself.
 
-What is the difference between a namespace package and a regular package?
-"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+Here is an example. Consider the package called ``diffpy.pdffit2``. The package starts with an ``org-name`` of ``diffpy``, and the package name is ``pdffit2``. There is a separate GitHub repository for this package (https://github.com/diffpy/diffpy.pdffit2) while it is developed under the DiffPy organization (https://github.com/diffpy). The user is able to import the package as ``import diffpy.pdffit2 as pdffit2`` in any Python script.
 
-The difference is in the folder structure:
+.. note::
+
+  This namespace feature is only available in Level 5, ``public``.
+
+What is the difference in folder structure compared to a standard package?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
-For a regular package ``bg-mpl-stylesheets``:
+For a regular package ``<project-name>``, the folder structure is the following:
 
 .. code-block:: text
 
   ├── src
-  │   ├── bg_mpl_stylesheets
+  │   ├── <project_name>
   │   │   ├── __init__.py
-  │   │   ├── colors.py
-  │   │   ├── inkscape_tools.py
+  │   │   ├── file_one.py
+  │   │   ├── file_two.py
 
-You would import it as ``import bg_mpl_stylesheets`` and the package name is ``bg-mpl-stylesheets`` on PyPI and conda-forge. See https://pypi.org/project/bg-mpl-stylesheets/.
-
-In constrat, here is an example of the ``diffpy.utils`` namespace package:
+For a package to be imported using ``import <org_name>.<package_name>``, here is the new structure:
 
 .. code-block:: text
 
-  ├── pyproject.toml
   ├── src
-  │   ├── diffpy
+  │   ├── <org_name>
```

Comment:
I wonder if we want <org_name> here because it doesn't have to be the org name. In our example it is, but in principle it doesn't have to be.  I wonder, since we already introduced the "namespace" nomenclature, we could in principle call it `<namespace_name>`.  We may not want to use that name for whatever reason, but also I think an org is different from this thing so it is probably not a good idea to use `org` in this context.